src/blr/blr.yaml

```diff
@@ -10,15 +10,20 @@ dbSNP: # Path to dbSNP file (common SNP sites)
 BQSR: false # Adjust base calling accuracys using gatk's BQSR (true or false). Recommended if variant_caller is gatk
 read_mapper: bowtie2  # Choose bwa, bowtie2, minimap2 or ema
 duplicate_marker: sambamba # Choose sambamba or samblaster
+library_type: blr # Library type. Chose blr or 10x.
```

Comment:
Any particular reason why you would prefer read_structure over library_type?

Reply:
The idea is that library type would decide read structure, where read structure is what decides how the reads should be trimmed. Granted in our situation all library types currently have a unique read structure, but I think calling it read structure is more to the point.

The problem I'm mainly seeing is a new user might not understand what implications having library type set to the wrong thing, whilst a faulty read structure would probably be quite self explanatory.

Reply:
Yeah maybe that is true. However it might also be harder for the user to get what they need to fill in as read_structure is a bit less intuitive, I think. Either way I think propper documentation will solve this ;) We could for example leave the default value out and instead change the messege in the `blr init` command to include "Please specify the library type / read structure in the blr.yaml file."